stk/calculators/energy/energy_calculators.py

```diff
@@ -377,3 +382,847 @@ def energy(self, mol, conformer=-1):
+        `conformer` pair.
+        Key is `(mol, conformer)`
+
+    full_quadrupole_moments : :class:`dict`
+        :class:`dict` of the `full` quadrupole moment of each `mol`,
+        `conformer` pair.
+        Key is `(mol, conformer)`
+
+    Examples
+    --------
+    .. code-block:: python
+
+        mol = StructUnit.smiles_init('NCCNCCN', ['amine'])
+        gfnxtb = XTBEnergy('/opt/gfnxtb/xtb')
+        gfnxtb.energy(mol)
+
+    Note that for :class:`.MacroMolecule` objects assembled by ``stk``
+    :class:`XTBEnergy` should usually be used after optimization with some
+    other method. This is because xTB only uses xyz coordinates as input
+    and so will not recognize the long bonds created during assembly.
+    An optimizer which can minimize these bonds should be used before
+    :class:`XTBEnergy`.
+
+    .. code-block:: python
+
+        bb1 = StructUnit2.smiles_init('NCCNCCN', ['amine'])
+        bb2 = StructUnit2.smiles_init('O=CCCC=O', ['aldehyde'])
+        polymer = Polymer([bb1, bb2], Linear("AB", [0, 0], 3))
+
+        uff = UFF()
+        uff.optimize(polymer)
+        xtb = XTBEnergy(xtb_path='/opt/gfnxtb/xtb',
+                            mem_ulimit=True)
+        xtb.energy(polymer)
+
+    Energies and other properties of optimized structures can be
+    extracted using :class:`XTBEnergy`. However, be aware that
+    :class:`XTBEnergy` will not check the quality of an optimized structure.
+
+    .. code-block:: python
+
+        xtb = OptimizerSequence(
+            UFF(),
+            XTB(xtb_path='/opt/gfnxtb/xtb',
+                mem_ulimit=True,
+                opt_level='normal',
+                solvent='THF')
+        )
+        xtb.optimize(polymer)
+
+        xtbenergy = XTBEnergy(xtb_path='/opt/gfnxtb/xtb',
+                              mem_ulimit=True,
+                              solvent='THF')
+        # runs calculation and returns energy
+        polymer_totalenergy = xtbenergy.energy(polymer, conformer)
+
+        # extracts properties from energy calculator for given conformer
+        polymer_homo_lumo_gap = xtbenergy.homo_lumo_gaps[(polymer, conformer)]
+        polymer_fermi_levels = xtbenergy.fermi_levels[(polymer, conformer)]
+        polymer_homo_lumo_orbitals = xtbenergy.homo_lumo_orbitals[(polymer, conformer)]
+        polymer_Qonly_dipole_moments = xtbenergy.Qonly_dipole_moments[(polymer, conformer)]
+        polymer_full_dipole_moments = xtbenergy.full_dipole_moments[(polymer, conformer)]
+        polymer_Qonly_quadrupole_moments = xtbenergy.Qonly_quadrupole_moments[(polymer, conformer)]
+        polymer_QDip_quadrupole_moments = xtbenergy.QDip_quadrupole_moments[(polymer, conformer)]
+        polymer_full_quadrupole_moments = xtbenergy.full_quadrupole_moments[(polymer, conformer)]
+
+        # the total energy can be extracted at any point from the calculator
+        polymer_totalenergy = xtbenergy.total_energies[(polymer, conformer)]
+
```

Comment:
I don't know if optimization should be discussed here. You could make the argument that before taking any energy with any calculator the structure should be optimized in some way. As a result I think it is redundant to mention this. Also, the properites and energies of any structures, not just optimized ones will be extracted. I would mentioned that this calculator can extract properties other than energy in the main section of the docstring as well, not just in the examples section.